.github/workflows/dev-stg-release.yml

```diff
@@ -0,0 +1,43 @@
+name: Dev/Staging BCentral Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      bal_central_environment:
+        description: Ballerina Central Environment
+        type: choice
+        options:
+        - STAGE
+        - DEV
+        required: true
+
+jobs:
+    release:
+
+        runs-on: ubuntu-latest
+        env: 
+          BALLERINA_${{ github.event.inputs.bal_central_environment }}_CENTRAL: true
+        steps:
+            - uses: actions/checkout@v2
+            - name: Ballerina Build
+              uses: ballerina-platform/ballerina-action/@2201.2.1
```

Comment:
Let's update this version with the specific Ballerina distribution u r planning to release from.
Define the specific version in the Ballerina.toml. Please apply the same in all the relevant places in the release.yml files
https://github.com/ballerina-platform/module-ballerinax-scim/blob/90c4d7d65893eb2e11a424cb2ca14a8f3ae9a600/scim/Ballerina.toml#L5